src/style.css

```diff
@@ -0,0 +1,23 @@
+body {
+  font-family: Arial, sans-serif;
+  font-size: 13px;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin-top: 8px;
```

Comment:
`The vertical spacing around "Preamble" and "Article N" must be 1em. All other vertical spacing must be set to zero, as in the document.`